source/Middlewares/Third_Party/FreeRTOS/Source/tasks.c

```diff
@@ -2093,6 +2100,10 @@ BaseType_t xReturn;
 	/* Prevent compiler warnings if INCLUDE_xTaskGetIdleTaskHandle is set to 0,
 	meaning xIdleTaskHandle is not used anywhere else. */
 	( void ) xIdleTaskHandle;
+
+	/* OpenOCD makes use of uxTopUsedPriority for thread debugging. Prevent uxTopUsedPriority
+	 * from getting optimized out as it is no longer used by the kernel. */
+	( void ) uxTopUsedPriority;

```

Comment:
Is this in upstream?

Reply:
Yes:
https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/534eba66ce4a5bda45d5edeeb81ac5a3cf6d0df8/tasks.c#L2108-L2110

Reply:
All good then.